CHANGELOG.md

```diff
@@ -3,6 +3,12 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [9.2.1] - 2023-05-19
```

Comment:
@pavel-kuznetsov-hypertrack you need to add a link to it at the bottom

Reply:
wdym?

Reply:
done